### PR TITLE
Account for more emptiness

### PR DESCRIPTION
**Pull request by hennevogel**
Let's not record empty strings...

### CODE REVIEW

**Comment by hennevogel** [comment]
It's weird and leads to strange queries that need regex etc. See on [OBS](https://obs-measure.opensuse.org/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22InfluxDB-Rails%22,%7B%22datasource%22:%22InfluxDB-Rails%22,%22policy%22:%22default%22,%22resultFormat%22:%22time_series%22,%22orderByTime%22:%22ASC%22,%22tags%22:%5B%7B%22key%22:%22hook%22,%22operator%22:%22%3D%22,%22value%22:%22process_action%22%7D%5D,%22groupBy%22:%5B%7B%22type%22:%22time%22,%22params%22:%5B%22$__interval%22%5D%7D,%7B%22type%22:%22fill%22,%22params%22:%5B%22null%22%5D%7D%5D,%22select%22:%5B%5B%7B%22type%22:%22field%22,%22params%22:%5B%22controller%22%5D%7D,%7B%22type%22:%22mean%22,%22params%22:%5B%5D%7D%5D%5D,%22measurement%22:%22rails%22%7D,%7B%22ui%22:%5Btrue,true,true,%22none%22%5D%7D%5D).

Screenshot from 2021-02-15 22-20-36

File: lib/influxdb/rails/tags.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def initialize(config:, tags: {}, additional_tags: InfluxDB::Rails.current.tags)
 
       def to_h
         expanded_tags.reject do |_, value|
-          value.nil? || value == ""
+          value.blank?
```
Comment (ChrisBr) [inline lib/influxdb/rails/tags.rb:12]: What is the difference?

Also IIRC there was some reason why we did not use blank 🤔

Comment (hennevogel) [reply lib/influxdb/rails/tags.rb:12]: `.blank?` covers strings of blank. Like: `'  '`

> Also IIRC there was some reason why we did not use blank thinking

Yeah https://github.com/influxdata/influxdb-rails/commit/6234e1c6f130280909254db09bf9fde90bfcea0f

**Comment by ChrisBr** [comment]
Okay that was stupid to merge this one as I remove it from master in https://github.com/influxdata/influxdb-rails/pull/160 🤦  I think we need to merge it again in the one branch.

**Pull request merged**